src/vm/i386/unixstubs.cpp

```diff
@@ -6,7 +6,11 @@
 
 extern "C"
 {
-    void ThrowControlForThread()
+    void ThrowControlForThread(
+#ifdef WIN64EXCEPTIONS        
```

Comment:
Actually, this PR does not include a commit to switch ``WIN64EXCEPTIONS``. (I applied it only internally). 

Is it required to remove #ifdef after adding ``WIN64EXCEPTIONS`` switch commit?